zerver/webhooks/clubhouse/tests.py

```diff
@@ -183,3 +194,10 @@ def test_story_update_type(self) -> None:
         expected_message = u"The type of the story [Add cool feature!](https://app.clubhouse.io/zulip/story/11) was changed from **feature** to **bug**."
         self.send_and_test_stream_message('story_update_change_type', "Add cool feature!",
                                           expected_message)
+
+    @patch('zerver.lib.webhooks.common.check_send_webhook_message')
+    def test_empty_post_request_body_ignore(self, check_send_webhook_message_mock: MagicMock) -> None:
```

Comment:
Are these for a `ping` event type?  If so, we should document that in the function name, comment, and/or docstring.

Reply:
@timabbott: Actually, I investigated and the Clubhouse webhook docs say nothing about a ping event type or an event type that sends an empty request. Their webhook API is still under active development, so it could just be an anomaly. I'll make this clear in a comment just for clarity! :)